frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -761,15 +784,56 @@ export const CRDCardRow = connect(crdCardRowStateToProps)((props: CRDCardRowProp
   );
 });
 
+const AlertToCreateCustomResource: React.SFC<AlertToCreateCustomResourceProps> = (props) => {
+  const { cr } = props;
+
+  const crName = cr?.[0]?.kind;
+  const crDefinition = props.obj.spec?.customresourcedefinitions?.owned?.find(
+    (api) => api.kind === crName,
+  );
+  const crModelName = crDefinition?.name;
+  const { group: crGroup } = groupVersionFor(cr?.[0]?.apiVersion);
+  const { findModel } = useModelFinder();
+  const model = findModel(crGroup, crModelName.split('.').shift());
+
+  // Check if the CR is already present - only checks for the specific name/ns in the initialization-resource
+  const [customResource, customResourceLoaded] = useK8sGet<K8sResourceKind>(
```

Comment:
Prefer `K8sResourceCommon` for unknown types. (It guards against trying to access `spec` and `status` values that might not be there.)

```suggestion
  const [customResource, customResourceLoaded] = useK8sGet<K8sResourceCommon>(
```